friedrich-schiller-university-jena-faculty-of-medicine.csl

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Friedrich Schiller University Jena - Faculty of Medicine</title>
+    <id>http://www.zotero.org/styles/friedrich-schiller-university-jena-faculty-of-medicine</id>
+    <link href="http://www.zotero.org/styles/friedrich-schiller-university-jena-faculty-of-medicine" rel="self"/>
+    <link href="http://www.zotero.org/styles/aquatic-conservation" rel="template"/>
+    <link href="http://www.uniklinikum-jena.de/MedWeb_media/Downloads/Dekanat_Promotionsstelle/Formulare/Zitierweise.pdf" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
+    <summary>Style for the medical faculty of the Friedrich-Schiller University Jena, derived primarily from Aquatic Conservation style</summary>
+    <updated>2016-12-16T17:09:57+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="et-al">et al.</term>
+    </terms>
+  </locale>
```

Comment:
@damnation333, this style is set to British English (`default-locale="en-GB"`), so this locale section will never be used. Assuming the "default-locale" is correct, could you delete this locale section?